plugins/linden.py

```diff
@@ -160,7 +160,9 @@ def invest_buy_cmd(user_id, args):
     if user_balance is None:
         return "User doesn't exist, get urself some Lindens"
     elif user_balance < ask * qt:
-        return "You don't have enough Lindens"
+        needs = ("You have {}L$ ".format(user_balance) +
```

Comment:
You could just do a `quote` after a failed buy. you'd need to refactor the code a bit though i think